src/python/grpcio/grpc/experimental/aio/_call.py

```diff
@@ -270,8 +275,12 @@ def __await__(self) -> ResponseType:
             if self._cython_call.is_locally_cancelled():
                 raise asyncio.CancelledError()
             else:
+                call_status = self._cython_call._status
+                debug_error_string = ""
+                if call_status is not None:
+                    debug_error_string = call_status._debug_error_string
                 raise _create_rpc_error(self._cython_call._initial_metadata,
-                                        self._cython_call._status)
+                                        call_status, debug_error_string)
```

Comment:
Comments regarding this piece of code are in this PR

[1] https://github.com/grpc/grpc/pull/21919